spell.ignore

```diff
@@ -1993,6 +1993,7 @@ vtype
 vv
 vvfat
 vvv
+vmcoreinfo
```

Comment:
Move this to the correct place(before `vmdk`), others LGTM.

Reply:
Updated, thanks~